spec/system/tasks_spec.rb

```diff
@@ -0,0 +1,134 @@
+require 'rails_helper'
+
+RSpec.describe "Tasks", type: :system do
+
+  let(:user) { create(:user) }
+  let(:task) { create(:task) }
+
+  describe 'ログイン前' do
+    describe 'ページ遷移確認' do
+      context 'タスクの新規登録ページにアクセス' do
+        it '新規作成ページへのアクセスが失敗する' do
+          visit new_task_path
+          expect(current_path).to eq login_path
+          expect(page).to have_content('Login required')
+        end
+      end
+
+      context 'タスクの編集ページにアクセス' do
+        it '編集ページへのアクセスが失敗する' do
+          visit edit_task_path(task)
+          expect(current_path).to eq login_path
+          expect(page).to have_content('Login required')
+        end
+      end
+
+      context 'タスクの詳細ページにアクセス' do
+        it 'タスクの詳細情報が表示される' do
+          visit task_path(task)
+          expect(current_path).to eq task_path(task)
+          expect(page).to have_content(task.title)
+        end
+      end
+
+      context 'タスクの一覧ページにアクセス' do
+        it 'すべてのユーザーのタスク情報が表示される' do
+          task_list = create_list(:task, 3)
+          visit tasks_path
+          expect(current_path).to eq tasks_path
+          expect(page).to have_content task_list[0].title
+          expect(page).to have_content task_list[1].title
+          expect(page).to have_content task_list[2].title
+        end
+      end
+    end
+  end
+
+  describe 'ログイン後' do
+
+    before { sign_in_as user }
+
+    describe 'タスク新規作成' do
+      context 'フォームの入力値が正常' do
+        it 'タスクの新規作成が成功する' do
+          visit new_task_path
+          fill_in 'task_title', with: 'test_title'
+          fill_in 'task_content', with: 'test_content'
+          select 'doing', from: 'Status'
```

Comment:
enumのプルダウンの選択は文字列ではなくSymbolの形式で実施した方が良いですね。
```suggestion
          select :doing, from: 'Status'
```

Reply:
e8ae10a4dbf7f4b7a94981466389b8acc0a8ec78 で文字列からシンボルに変更しました。